src/main/java/play/socketio/javadsl/SocketIOEventCodec.java

```diff
@@ -276,11 +275,11 @@
      * @param args The arguments to decode.
      * @return The decoded event.
      */
-    T decodeArgs(Seq<Either<JsValue, ByteString>> args);
+    T decodeArgs(List<Either<JsValue, ByteString>> args);
```

Comment:
Define everything with `java.util.List` instead of Seq as the type is different between 2.12 / 2.13

Reply:
Sounds reasonable to me.